models/Event.js

```diff
@@ -501,4 +516,10 @@ Event.beforeValidate(async (event) => {
     if (typeof event.url === 'string') event.url = event.url.toLowerCase().trim();
 });
 
+Event.afterSave(async (event) => {
+    if (event.available_spots <= 0 && event.open_call === true) {
```

Comment:
it can be negative?

Reply:
It shouldn't, so maybe I should change it to === 0

Reply:
Fixed